communications/ff_msgs/msg/PerchState.msg

```diff
@@ -43,13 +44,14 @@ int8 PERCHING_WAITING_FOR_SPIN_DOWN     = 3
 int8 PERCHING_SWITCHING_TO_PL_LOC       = 2
 int8 PERCHING_STOPPING                  = 1
 int8 PERCHED                            = 0
-int8 UNPERCHING_SWITCHING_TO_HR_LOC     = -1
-int8 UNPERCHING_WAITING_FOR_SPIN_UP     = -2
-int8 UNPERCHING_OPENING_GRIPPER         = -3
-int8 UNPERCHING_MOVING_TO_APPROACH_POSE = -4
-int8 UNPERCHING_STOWING_ARM             = -5
-int8 UNPERCHING_SWITCHING_TO_ML_LOC     = -6
-int8 UNPERCHED                          = -7
+int8 UNPERCHED                          = -1
```

Comment:
Unfortunately, this still breaks some assumptions in the executive and GDS. I think this may just have to be something an operator looks up.

Reply:
Alright then :+1: